cdap-ui/app/tracker/services/my-jump-factory.js

```diff
@@ -53,20 +47,23 @@ function myJumpFactory($state, myTrackerApi, $rootScope) {
     }
   };
 
-  let realtimeSinkTemplate = {
-    artifact: realtimeArtifact,
-    config: {
-      source: {},
-      transforms: [],
-      connections: []
-    }
-  };
-
   let cdapDatasetTypes = {
     Table: 'co.cask.cdap.api.dataset.table.Table',
     KVTable: 'co.cask.cdap.api.dataset.lib.KeyValueTable'
   };
 
+  function _redirect(data) {
+    let url = window.getHydratorUrl({
+      stateName: 'hydrator.create',
+      stateParams: {
+        namespace: $state.params.namespace,
+        configParams: data
+      }
+    });
+
+    window.location.replace(url);
```

Comment:
This will replace the history. So if the user goes from hydrator -> tracker and back to hydrator (to add as source) this will not record going to tracker in history. So hitting back from the last step in the browser will take the user back to hydrator instead of tracker. Should probably use `window.location.href = url`